src/API/Logs/EventLoggerInterface.php

```diff
@@ -4,10 +4,24 @@
 
 namespace OpenTelemetry\API\Logs;
 
+use OpenTelemetry\Context\ContextInterface;
+
 /**
  * @see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md#events-api-interface
  */
 interface EventLoggerInterface
 {
+    public function emit(
+        string $name,
+        mixed $payload = null,
+        ?int $timestamp = null,
+        ?ContextInterface $context = null,
+        Severity|int|null $severityNumber = null,
+        ?array $attributes = [],
+    ): void;
+
+    /**
+     * @deprecated Use `emit`
+     */
     public function logEvent(string $eventName, LogRecord $logRecord): void;
```

Comment:
IMO it doesn't make sense to keep this method, the definition of the `event.name` attribute was changed in SemConv 1.24 -> previous calls to `::logEvent()` might populate the attribute with an incorrect value:
> BREAKING: Change event.name definition to include namespace and remove event.domain from log event attributes.